install/gcp/installer/forseti_server_installer.py

```diff
@@ -232,8 +258,14 @@ def determine_access_target(self):
         """
         utils.print_banner('Forseti Installation Configuration')
 
-        self.access_target = constants.RESOURCE_TYPES[0]
-        self.target_id = self.organization_id
+        if self.composite_root_resources:
```

Comment:
i am a bit unclear on this function, but should it not be granting access to all the composite roots instead of just the first?

Reply:
The role granting does occur to all composite resources later. Initially I too question the value of this block. I'll revisit this later and see if it is still relevant with composite.